src/main/java/com/aesopwow/subsubclipclop/domain/api/service/ApiService.java

```diff
@@ -14,4 +14,6 @@ public interface ApiService {
     public ApiResponseDto requestAnalysis(ApiRequestDto apiRequestDto);
 
     public byte[] getAnalysisResult(String filename);
+
+    public byte[] getAnalysisResult2(String infoDbNo, String originTable);
```

Comment:
```suggestion
    public byte[] getAnalysisResult(String infoDbNo, String originTable);
```
위 내용이 필요없다면 지우고 만드는게 더 좋아보임 굳이 2를 붙여 더 헷갈리게 만들 필요가 있나 싶음